protocols/v2/framing-sv2/src/framing.rs

```diff
@@ -83,16 +81,18 @@ impl<T: Serialize + GetSize, B: AsMut<[u8]> + AsRef<[u8]>> Sv2Frame<T, B> {
     /// already serialized payload. If the frame has not yet been
     /// serialized, this function should never be used (it will panic).
```

Comment:
we should update these comments so that they are coherent with the new behavior of this function

updating the comments is especially important given that these are used on the Rust Docs, so they need to follow these changes accordingly